src/ALB.Api/UseCases/Endpoints/Admin/ApplicationUser/ReadApplicationUser/ReadApplicationUserEndpoint.cs

```diff
@@ -0,0 +1,19 @@
+using FastEndpoints;
+
+namespace ALB.Api.UseCases.Endpoints.Admin.ApplicationUser.ReadApplicationUser;
+
+public class ReadApplicationUserEndpoint : Endpoint<ReadApplicationUserRequest, ReadApplicationUserResponse>
+{
+    public override void Configure()
+    {
+        Get("/admin/application-user/read-application-user");
```

Comment:
Das Pfad in dem Get() steht für eine Liste aller User. Wenn man einen einzelnen möchte muss da die ID noch mit rein. Siehe suggestion.

```suggestion
        Get("api/users/{id}");
```